Error rerunning make generate on Linux

From a freshly cloned repository. Ran the following:

```
cd ui/v2
yarn install
cd../..
make generate
```
This generates the files correctly. Note: the fact that `yarn install` needs to be run before `make generate` for the first time should probably be documented in the readme.

Now, run `make generate` again and the following output is displayed:

```
go generate
modelgen: go [list -e -json -compiled=true -test=false -export=false -deps=true -find=false -- github.com/stashapp/stash/pkg/models github.com/99designs/gqlgen/graphql/introspection github.com/99designs/gqlgen/graphql]: exit status 1: go build <src_dir>/stash/packrd: no Go files in

exit status 3
main.go:1: running "go": exit status 1
make: *** [Makefile:18: generate] Error 1
```

Performing the following allowed `make generate` to function again:

```
rm -rf packrd pkg/api/api-packr.go pkg/database/database-packr.go
```

I have had no such issue on Windows.

The issue appears to be caused by [this line in main.go](https://github.com/stashapp/stash/blob/a2e550063bd0fc12c809d9a931cd5f179f07d223/main.go#L2) introduced in #132 :

```
//go:generate go run github.com/gobuffalo/packr/v2/packr2
```

Removing this line fixed the issue, and I suspect that this shouldn't be included here anyway. `make generate` is intended to be run before running `make ui` which is intended to be run before `make build`, which is where `packr` is actually ran.

--- a/main.go
+++ b/main.go
@@ -1,5 +1,4 @@
 //go:generate go run github.com/99designs/gqlgen
-//go:generate go run github.com/gobuffalo/packr/v2/packr2
 package main
 
 import (

